site/gdocs/centered-article.scss

```diff
@@ -18,6 +18,19 @@ $banner-height: 200px;
     z-index: -1;
 }
 
+// Until we implement a way to specifiy src-sets in archie, this image will need to have a 1:1 aspect ratio
+// to fit on both mobile and desktop resolutions
```

Comment:
I don't quite get this 1:1 ratio constraint. Whether on mobile or desktop, the image is never quite square or meant to be, is it?

I also don't get why we would want multiple cover images.

Reply:
True, 1:1 was just an approximation of the smallest view that the site reasonably supports:

<img width="335" alt="image" src="https://github.com/owid/owid-grapher/assets/11844404/78d95b8e-bae2-4608-9b88-e637529eac93">

And so eventually I imagine we'd want to be able to design one cover image to use when the aspect ratio is like that, and a separate one to use on wider screens that would be more rectangular.

Of course, we could stick to using the same asset in both cases and just center and scale it - it depends on what kind of designs we want to use for assets here. I'm kicking that can with this PR until we actually have articles that we want to cover.